apps/crn-server/test/handlers/webhooks/fetch-by-code/fetch-by-code.test.ts

```diff
@@ -151,7 +151,12 @@ describe('Fetch-user-by-code handler', () => {
       )) as APIGatewayProxyResult;
 
       expect(result.statusCode).toStrictEqual(200);
-      expect(JSON.parse(result.body)).toMatchObject(getUserResponse());
+
+      // Using stringify() in the expected response since the result body
+      // is also using it and undefined properties are removed when running it.
+      const expected = JSON.parse(JSON.stringify(getUserResponse()));
```

Comment:
you say that this needs to be done because of undefined properties but `toMatchObject` should allow you to have them